app/src/main/java/com/example/campuscollab/service/UserService.java

```diff
@@ -122,12 +145,50 @@ private User mapDocToUser(DocumentSnapshot documentSnapshot) {
         user.setDescription((String) documentSnapshot.get("description"));
         user.setLinkedInUrl((String) documentSnapshot.get("linkedInUrl"));
         user.setMajor((String) documentSnapshot.get("major"));
-        user.setPictureUrl((String) documentSnapshot.get("pictureUrl"));
+        user.setImagePath((String) documentSnapshot.get("imagePath"));
         user.setSkills((List<String>) documentSnapshot.get("skills"));
         user.setResumeUrl((String) documentSnapshot.get("resumeUrl"));
         user.setCreatedDate((Timestamp) documentSnapshot.get("createdDate"));
+        user.setSchool((String) documentSnapshot.get("school"));
 
         return user;
     }
 
+    public AsyncTask<Void, Void, byte[]> getImageBytes(String imagePath) {

```

Comment:
@CaraHamilton I know Kai was working on implementing some of this, have you two been coordinating this?

Reply:
Disregard. Looks like these are Kai's changes got merged that somehow these are carrying over into your pr. Sorry!